model_analyzer/config/input/config_command_analyze.py

```diff
@@ -369,6 +380,18 @@ def _autofill_values(self):
                         'max': self.latency_budget
                     }})
 
+        # Set global constraints if latency budget is specified
+        if self.min_throughput:
```

Comment:
if `min_throughput` is specified and equal to zero this will not work.

Reply:
Im thinking about it, and if the min throughput is 0, and this branch is not taken, its equivalent to not having the constraint since the throughput must be positive.